### PR TITLE
Math Magician: Components

### PR DESCRIPTION
**Pull request by anyars-encarta**
In this project:

- [ ] I deleted the boilerplate from the Create React App (text, images, styles).
- [ ] I created a new directory called components.
- [ ] Inside components, I created a new `Calculator.js` file.
- [ ] In `Calculator.js`, I created a React component that matches the required Calculator design:
- [ ] I Implemented the design using only JSX and CSS.
- [ ] I used functional components
- [ ] I created at least 1 child component
- [ ] The Calculator was imported into the `App.js` file
- [ ] The app renders in the browser without problems.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        <button type="button">2</button>
+        <button type="button">3</button>
+        <button type="button" className="side">+</button>
+
+        <button type="button" className="zero">0</button>
+        <button type="button">.</button>
+        <button type="button" className="side">=</button>
+      </div>
+    </div>
+  );
```
Comment (ShadyShawkat) [inline src/components/Calculator.js]: - [ ] Kindly make sure that this component has at least one child component, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Create%20at%20least%201%20child%20component) requirement.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @anyars-encarta, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (anyars-encarta) [reply src/components/Calculator.js]: Thanks. I will do that asap

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @anyars-encarta 

Good job implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitFlow 
-  ✅ Child component is used in `Calculator.js`
-  ✅ Linters are working properly

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**